app/src/main/res/navigation/nav_graph.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/nav_graph"
+    app:startDestination="@id/stackFragment">
+
+    <fragment
+        android:id="@+id/stackFragment"
+        android:name="com.ataulm.stacks.StackFragment"
+        android:label="fragment_stack"
+        tools:layout="@layout/fragment_stack">
+        <action
+            android:id="@+id/action_stackFragment_self"
+            app:destination="@id/stackFragment" />
+        <argument
+            android:name="stackId"
+            app:argType="string"
+            app:nullable="true"
+            android:defaultValue="@null" />
```

Comment:
stackId is nullable because the top level stacks have no parent.

We could initialise all top level stacks with an ID like -1 or something fixed, but I can't think of a compelling reason.